get-started/resources.md

```diff
@@ -0,0 +1,68 @@
+    * [Java Development: Netbeans](https://training.play-with-docker.com/java-debugging-netbeans/)
+    * [Live Debugging Node.js with Docker and Visual Studio Code](https://training.play-with-docker.com/nodejs-live-debugging/)
+
+
+* **Windows Containers**
+    * [Windows Container Setup](https://training.play-with-docker.com/windows-containers-setup/)
+    * [Windows Container Basics](https://training.play-with-docker.com/windows-containers-basics/)
+    * [Windows Containers Multi-Container Applications](https://training.play-with-docker.com/windows-containers-multicontainer/)
+
+# Books
+If books are your preferred learning style, check out these from Docker Architect Elton Stoneman:
+* [Learn Docker in a Month of Lunches (use the code `webdoc40` for 40% off)](https://www.manning.com/books/learn-docker-in-a-month-of-lunches)
+* [Docker on Windows: From 101 to Production with Docker on Windows](https://www.amazon.com/Docker-Windows-Elton-Stoneman-ebook/dp/B0711Y4J9K/)
+
+Or these written by the [Docker Captains](https://www.docker.com/community/captains). Docker Captain is a distinction that Docker awards to select members of the community that are both experts in their field and are committed to sharing their Docker knowledge with others.
+
+* [The Kubernetes Book](https://www.amazon.com/Kubernetes-Book-Nigel-Poulton/dp/1521823634/ref=sr_1_3?ie=UTF8&qid=1509660871&sr=8-3&keywords=nigel+poulton), Nigel Poulton, Nov 2018
+* [Docker Deep Dive](https://www.amazon.com/Docker-Deep-Dive-Nigel-Poulton/dp/1521822808/ref=sr_1_1?ie=UTF8&qid=1509660871&sr=8-1&keywords=nigel+poulton), Nigel Poulton, March 2018
+* [Docker in Action](https://www.amazon.com/Docker-Action-Jeff-Nickoloff/dp/1633430235/)  **Updated (2018)** Jeff Nickoloff
+* [Portuguese] [Docker para desenvolvedores](https://leanpub.com/dockerparadesenvolvedores) (2017) by Rafael Gomes
+
+# Self-Paced Online Learning
+A number of Docker Captains have also created video courses on Docker and Kubernetes.
+
+* [Bret Fisher](https://www.bretfisher.com/courses/): Docker Mastery, Docker Swarm Mastery, Docker Mastery for Node.js Projects
+* [Nick Janetakis](https://nickjanetakis.com/courses/) Dive into Docker, Docker for DevOps
+* [Nigel Poulton](https://nigelpoulton.com/video-courses): Kubernetes 101, Getting Started with Kubernetes, Docker and Kubernetes: The Big Picture, Kubernetes Deep Dive, Docker Deep Dive
+* [Arun Gupta](https://www.lynda.com/Docker-tutorials/Docker-Java-developers/576584-2.html): Docker for Java Developers
+* [Ajeet Singh Raina](https://collabnix.com/): Docker and Kubernetes Labs
+* [French] [Luc Juggery](https://www.udemy.com/user/lucjuggery/): Introduction to Kubernetes, The Docker Platform
+* [Spanish] [Manuel Morejón](https://www.udemy.com/course/integrando-docker-a-su-infraestrucutra-y-servicios/learn/lecture/6624848#overview): Integrating Docker to its Infrastructure and Services
+
+\* Many of the courses are fee-based
```

Comment:
Same comment about the asterisk. Is this to distinguish the free courses from the fee-based courses?

Reply:
Good point. I was not thinking about the way the markdown renders.